src/ducks/apps/components/Sections/components/AppsSection.jsx

```diff
@@ -18,6 +18,8 @@ const _renderAppComponent = (app, t, onAppClick, isMobile) => {
   )
 }
 
+const appSorter = t => app => getTranslatedManifestProperty(app, 'name', t)
```

Comment:
nit: I would rename this `makeNameGetter` since this is clearly returning a getter and not a sorter here

Reply:
Done in https://github.com/cozy/cozy-store/pull/622/commits/5636fdb6108fedb74b5051ec0970fded22f4ef27